client/state/sharing/publicize/selectors.js

```diff
@@ -52,6 +52,19 @@ export function getSiteUserConnectionsForService( state, siteId, userId, service
 }
 
 /**
+ * Returns true when there are broken connections for the specified service.
+ *
+ * @param  {Object} state   Global state tree
+ * @param  {Number} siteId  Site ID
+ * @param  {Number} userId  User ID to filter
+ * @param  {String} service The name of the service to check
+ * @return {Array}          Broken user connections.
+ */
+export function getBrokenSiteUserConnectionsForService( state, siteId, userId, service ) {
```

Comment:
Right, though this returns the actual connections so they can be fixed. I did take over your test fixes though